src/main/java/edu/jhuapl/dorset/demos/DBManager.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2017 The Johns Hopkins University Applied Physics Laboratory LLC
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.jhuapl.dorset.demos;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+import org.hibernate.Query;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.criterion.Restrictions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DBManager implements ToDoListManager {
+    private static final Logger logger = LoggerFactory.getLogger(DBManager.class);
+
+    private SessionFactory factory;
+
+    /**
+     * Create a DB Manager
+     *
+     * @param toDoListName   the name of ...TODO
+     */
+    public DBManager(String toDoListName) {
+        //TODO toDoListName not used
+        Configuration configuration = new Configuration().configure();
```

Comment:
Are you planning on using the toDoListName? If not, you should probably remove from the constructor.